sqlx-mysql/src/testing/mod.rs

```diff
@@ -52,22 +52,6 @@ impl TestSupport for MySql {
         })
     }
 
-    fn cleanup_test_dbs() -> BoxFuture<'static, Result<Option<usize>, Error>> {
-        Box::pin(async move {
-            let url = dotenvy::var("DATABASE_URL").expect("DATABASE_URL must be set");
-
-            let mut conn = MySqlConnection::connect(&url).await?;
-
-            let now = SystemTime::now()
-                .duration_since(SystemTime::UNIX_EPOCH)
-                .unwrap();
-
-            let num_deleted = do_cleanup(&mut conn, now).await?;
-            let _ = conn.close().await;
-            Ok(Some(num_deleted))
-        })
-    }
-
```

Comment:
Reminder to check this trait implementation for proper quoting of the database name. It doesn't look like it does it.

Reply:
@bonega don't forget this. Even with `URL_SAFE` the database name still needs to be quoted because it may contain dashes.

Reply:
I checked and it seems to work.

A test in `examples/postgres/axum-social-with-tests` has the `test_path`: `comment::test_create_comment`
Which becomes the database name of `_sqlx_test_hsAQWz-87IRR7sjnVDeMcHtoDLU3QyT6yWizSbWKFvNwoBt6Q60I`

It is deleted correctly even though it contains dashes.

I see now that you're referencing the mysql implementation which I haven't started yet.
I will try and do it and the sqlite today.

Reply:
Just to update that I am fighting with running the tests on (both my branch and main)